rabbitmq/resource_shovel.go

```diff
@@ -257,6 +255,38 @@ func ReadShovel(d *schema.ResourceData, meta interface{}) error {
 	return nil
 }
 
+func UpdateShovel(d *schema.ResourceData, meta interface{}) error {
+	rmqc := meta.(*rabbithole.Client)
+
+	ID := strings.Split(d.Id(), "@")
+	if len(ID) < 2 {
+		return fmt.Errorf("Unable to determine Shovel ID")
+	}
+
+	shovelName := ID[0]
+	vhost := ID[1]
```

Comment:
You cannot read the name and vhost from the ID, because of that updating name or vhost does not work anymore.

Also, as they are part of the ID, I'd keep them with `ForceNew` even if we support update for info.

Reply:
It looks like this technique of parsing the resource id is already used in all other "update" methods in the provider (operator_policy, permissions, policy, topic_permission)

I think this could stay as-is as long as we set ForceNew on change of vhost and name so that the update method is not called in those cases.

What do you think ?